.werft/jobs/build/validate-changes.ts

```diff
@@ -29,7 +29,7 @@ async function branchNameCheck(werft: Werft, config: JobConfig) {
 
 async function preCommitCheck(werft: Werft) {
     werft.log("pre-commit checks", "Running pre-commit hooks.")
-    const preCommitCmd = exec(`pre-commit run --all-files --show-diff-on-failure`, { slice: "pre-commit checks" });
+    const preCommitCmd = exec(`pre-commit run --show-diff-on-failure`, { slice: "pre-commit checks" });
```

Comment:
IMO, there's limited value in running pre-commit on all files. Not only does it prevent incremental incremental lint fix, it also takes longer.

Reply:
Fine with that. I don't think we have a case of "transitive invalidation" in any of our checks,